src/engraving/rw/read410/read410.cpp

```diff
@@ -156,8 +156,8 @@ bool Read410::readScore410(Score* score, XmlReader& e, ReadContext& ctx)
             score->m_showInvisible = e.readInt();
         } else if (tag == "showUnprintable") {
             score->m_showUnprintable = e.readInt();
-        } else if (tag == "showFrames") {
```

Comment:
Do these changes take into account the possibility of opening old files whose elements (tags) have old names?
I apologize in advance if my comments are out of place.

Reply:
I believe it should read both tags

Reply:
actually that change should get dropped